src/util/message.h

```diff
@@ -101,4 +101,18 @@ uint256 MessageHash(const std::string& message, MessageSignatureFormat format);
 
 std::string SigningResultString(const SigningResult res);
 
+/**
+ * Generate the BIP-322 tx corresponding to the given challenge
+ */
+class BIP322Txs {
```

Comment:
nit: Default visibility for classes in C++ is private, but IMHO it would still be more readable if that is stated explicitly:
```suggestion
class BIP322Txs {
private:
```